test/built-ins/Promise/prototype/finally/invokes-then-with-function.js

```diff
@@ -27,6 +27,7 @@ target.then = function(a, b) {
 };
 
 var originalFinallyHandler = function () {};
+var anonName = Object(function () {}).name;
```

Comment:
The `Object()` call here is to avoid the function name inference pitfall; and pulling it off of a new function is to avoid having to hardcode the logic for an anonymous function name in this test (that should be tested elsewhere).

Reply:
you're assigning the value of `.name` to `anonName`. I wonder if you can just use the empty string directly.

In any case, this won't make the test wrong.

Reply:
There's some browsers where it's "anonymous"; I wanted to insulate this test against that unrelated failure.